variables.yaml.example

```diff
@@ -1,39 +1,44 @@
+google:
+  # The environment you would like to provision e.g. dev
+  # This should be the full project id if it is an existing project.
+  project_name: ''
+  folder_id: ''
+  billing_account: ''
+  region: 'europe-west2'
+  machine_type: 'n1-standard-2'
```

Comment:
`machine_type` doesn't look right here. The rest of the variables are all top level gcp things, this is specific to the machine type used for kubernetes pods. I think this should be defined outside of the `google` property.